src/validationinterface.h

```diff
@@ -114,7 +114,7 @@ class CValidationInterface {
      *
      * Called on a background thread.
      */
-    virtual void BlockDisconnected(const std::shared_ptr<const CBlock> &block) {}
+    virtual void BlockDisconnected(const std::shared_ptr<const CBlock>& block, const CBlockIndex* pindex) {}
```

Comment:
3a04cd6776f7f1ec86cad20de90c9fbb24655536

nit, could drop `&` from 1st argument, it was discussed elsewhere that smart pointers and iterators should be used as value.

Reply:
Please don't do this. Trying to 'fix' surrounding code in a PR makes the diff bigger and more confusing for reviewers.